wsflate/cbuf.go

```diff
@@ -0,0 +1,93 @@
+package wsflate
+
+import (
+	"io"
+)
+
+// cbuf is a tiny proxy-buffer that writes all but 4 last bytes to the
+// destination.
+type cbuf struct {
```

Comment:
what's the reason to drop the last 4 bytes?

Reply:
It's by [the spec](https://tools.ietf.org/html/rfc7692#section-7.2.1).